### DISCUSSION TITLE
Error Logout ...

### DISCUSSION THREAD

**Discussion by mildblue** [rejetto/hfs#676] [Q&A]
![2024-07-10_090150](https://github.com/rejetto/hfs/assets/13603473/098e1bd7-90da-416d-8304-811b5e76b3f4)
![2024-07-10_090205](https://github.com/rejetto/hfs/assets/13603473/7d03877d-e702-4736-a7a4-a31cd2e5709e)

You are not logout.
It says 'Error'.

**Comment by rejetto** [comment]
can you press F12 and look in the "network" tab, to look at the content of the "logout" request?

**Reply by mildblue** [reply]
> can you press F12 and look in the "network" tab, to look at the content of the "logout" request?

![2024-07-10_202849](https://github.com/rejetto/hfs/assets/13603473/b4a574ac-f5f7-4b67-8bcc-a8f05189e3a2)
![2024-07-10_203150](https://github.com/rejetto/hfs/assets/13603473/809ea158-dcb4-4566-8056-dccd02b1b049)

**Comment by rejetto** [comment]
thanks!
i still need to see some important content, not visible in your screenshots.
inside the network, you should click on the red "logout", and then look into "response", like this
<img width="554" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/b945eef0-1f03-46b6-a1aa-5b81198a4005">

are you using nginx or other proxy?
